docs/extensibility/form-section.md

```diff
@@ -319,7 +319,7 @@ Resource widgets render a dropdown list of specified resources and store the sel
 - **filter** - JSONata expression that filters resources based on a given condition.
 - **provideVar** - when this field is defined, the chosen resource is provided as a variable of this name.
 - **required** - a boolean which specifies if a field is required. The default value is taken from CRD; if it doesn't exist in CRD, then it defaults to `false`.
-- **inputInfo** - a string below the input field that shows how to fill in the input.
+- **inputInfo** - a string below the input field that shows how to fill in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.
```

Comment:
```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.
```